app.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# project template
+# Copyright(C) 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Graph backup logic for the Thoth project."""
+
+import logging
+import os
+import subprocess
+from datetime import datetime
+from thoth.storages import __version__ as __storages__version__
+from thoth.storages import DatabaseBackupStore
```

Comment:
Please use `init_logging` from `thoth.common` so that exceptions produced are reported to Sentry.

Reply:
Thanks for pointing it out, will add env vars required in the openshift templates.